project_event/models/project_task.py

```diff
@@ -871,16 +890,13 @@ def child_reservation(child):
     def get_message_body(action):
         switcher = {
             'draft': ' ',
-            'option': _('The following is optional and \
-                        appears as crosshatched on your calendar'),
-            'requested': _('The following is requested'),
-            'accepted': _('The following is approved'),
+            'option': _(MSG_OPTION),
```

Comment:
Did you check this works for translations? @jananjoy

Reply:
@rimbendhaou  it works